### ISSUE TITLE
Flip or Flop not working

### ISSUE THREAD

**Issue by randy3k** [learningit/plugin.video.hgtv#2]
```
14:09:30 T:4622651392  NOTICE: -->Python Interpreter Initialized<--
14:09:30 T:4622651392   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.ValueError'>
                                            Error Contents: unknown url type: 
                                            Traceback (most recent call last):
                                              File "/Users/Randy/Library/Application Support/Kodi/addons/plugin.video.hgtv/default.py", line 225, in <module>
                                                elif mode=='GE':  getEpisodes(p('url'), p('name'))
                                              File "/Users/Randy/Library/Application Support/Kodi/addons/plugin.video.hgtv/default.py", line 141, in getEpisodes
                                                html = getRequest(url)
                                              File "/Users/Randy/Library/Application Support/Kodi/addons/plugin.video.hgtv/default.py", line 55, in getRequest
                                                response = urllib2.urlopen(req, timeout=30)
                                              File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.10_x86_64-target/lib/python2.6/urllib2.py", line 126, in urlopen
                                              File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.10_x86_64-target/lib/python2.6/urllib2.py", line 383, in open
                                              File "/Users/Shared/jenkins/workspace/OSX-64/tools/depends/xbmc-depends/macosx10.10_x86_64-target/lib/python2.6/urllib2.py", line 244, in get_type
                                            ValueError: unknown url type: 
                                            -->End of Python script error report<--
14:09:30 T:140735294051072   ERROR: GetDirectory - Error getting plugin://plugin.video.hgtv/?mode=GE&name=Flip%20or%20Flop&url=http%3a%2f%2fwww.hgtv.com%2fshows%2fflip-or-flop%2fflip-or-flop-full-episodes-videos
14:09:30 T:140735294051072   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.hgtv/?mode=GE&name=Flip%20or%20Flop&url=http%3a%2f%2fwww.hgtv.com%2fshows%2fflip-or-flop%2fflip-or-flop-full-episodes-videos) failed
```

2.1.4b

**Comment by learningit** [comment]
1) Post a full debug log to either xbmlogs.om or pastebin.com
2) post a link and problem description to the kodi support thread for the addon:
http://forum.xbmc.org/showthread.php?tid=97018

The partial log you give above is useless.

**Comment by learningit** [comment]
New version fixes when the website returns a null url for a video. Why it does that, I don't know.

**Comment by learningit** [comment]
New version 2.1.5 fixes when the website returns a null url for a video. Why it does that, I don't know.
This should fix the issue #2 above
